x-pack/test/security_solution_cypress/config.ts

```diff
@@ -55,6 +55,11 @@ export default async function ({ readConfigFile }: FtrConfigProviderContext) {
         `--home.disableWelcomeScreen=true`,
         // Specify which version of the detection-rules package to install
         // `--xpack.securitySolution.prebuiltRulesPackageVersion=8.3.1`,
+        // Set an inexistent directory as the Fleet bundled packages location
+        // in order to force Fleet to reach out to the registry to download the
+        // packages listed in fleet_packages.json
+        // See: https://elastic.slack.com/archives/CNMNXV4RG/p1683033379063079
+        `--xpack.fleet.developer.bundledPackageLocation=./inexistentDir`,
```

Comment:
Since we added a test in this PR that asserts that the `security_detection_engine` is downloaded correctly from Fleet (from the registry and not from the bundled packages created during build time), we need to set this config to an inexistent directory, so the installation fails and Fleet fetches the package from EPR. See the Slack thread pasted in the code for reference.

Also FYI @MadameSheema